packages/renderer/src/AppNavigation.svelte

```diff
@@ -49,7 +50,7 @@ export let meta;
 
 <svelte:window bind:innerWidth="{innerWidth}" />
 <nav
-  class="pf-c-nav z-0 group w-12 hover:w-[180px] md:w-[180px] md:min-w-[180px] shadow flex-col justify-between sm:flex transition-all duration-500 ease-in-out"
+  class="pf-c-nav z-0 group w-14 hover:w-[250px] md:w-[250px] hover:sm:w-[250px] md:min-w-[200px] shadow flex-col justify-between sm:flex transition-all duration-500 ease-in-out overflow-hidden hover:overflow-y-auto"
```

Comment:
It increases the width of the left navbar no ?

Reply:
Width of collapsed left bar was increased to have equal spacing on the icons:

<details>
  <summary>Before update:</summary>

![Monosnap Podman Desktop 2023-02-09 12-42-10](https://user-images.githubusercontent.com/1968177/217793579-df264cd5-c571-4f39-8fde-18e57e806869.jpg)
</details>

<details>
  <summary>After setting width to 3.5rem:</summary>

![Monosnap Podman Desktop 2023-02-09 12-37-22](https://user-images.githubusercontent.com/1968177/217793956-abf1f640-ce92-4c8d-86ea-6544f43594b7.jpg)

</details>

Width of the left navigation bar was increased a bit to not to have issues with displaying extension list:

<details>
  <summary>Before update(default opening state):</summary>

![Monosnap Podman Desktop 2023-02-09 12-48-03](https://user-images.githubusercontent.com/1968177/217794448-1bbeae7b-5613-4c63-88f1-729385946020.jpg)

</details>

<details>
  <summary>After setting new width values(default opening state):</summary>

![Monosnap Podman Desktop 2023-02-09 12-39-55](https://user-images.githubusercontent.com/1968177/217794610-689b6cb1-99f6-4b63-9d03-de4e55623976.jpg)

</details>


<details>
  <summary>Before update(scaled window to 1300px):</summary>

![Monosnap Podman Desktop 2023-02-09 12-46-40](https://user-images.githubusercontent.com/1968177/217795089-aea54357-159e-4661-8529-7f39a263ec80.jpg)

</details>


<details>
  <summary>After update(scaled window to 1300px):</summary>

![Monosnap Podman Desktop 2023-02-09 12-40-24](https://user-images.githubusercontent.com/1968177/217795294-c6feef2a-6fb5-410e-80c6-9bbf67edd30f.png)

</details>

Visually, difference is little, but extension list looks prettier.